packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -29,88 +30,109 @@ class DebugSessions extends StatelessWidget {
           const Text('Begin a debug session to use DevTools.')
         else
           Table(
-            columnWidths: const {0: FlexColumnWidth()},
-            defaultColumnWidth:
-                FixedColumnWidth(defaultIconSize + defaultSpacing),
+            columnWidths: const {
+              0: FlexColumnWidth(2),
+              // TODO(dantup): Fixed width icons+menu?
+              1: FlexColumnWidth(),
+            },
             defaultVerticalAlignment: TableCellVerticalAlignment.middle,
             children: [
               for (final session in sessions)
-                _createSessionRow(context, session),
+                TableRow(
+                  children: [
+                    Text(
+                      '${session.name} (${session.flutterMode})',
+                      style: Theme.of(context).textTheme.titleSmall,
+                    ),
+                    if (api.capabilities.openDevToolsPage)
+                      _DevToolsMenu(api: api, session: session),
+                  ],
+                ),
             ],
           ),
       ],
     );
   }
+}
+
+class _DevToolsMenu extends StatelessWidget {
+  const _DevToolsMenu({required this.api, required this.session});
+
+  final VsCodeApi api;
+  final VsCodeDebugSession session;
 
-  TableRow _createSessionRow(BuildContext context, VsCodeDebugSession session) {
+  @override
+  Widget build(BuildContext context) {
     // TODO(dantup): What to show if mode is unknown (null)?
-    final name = session.name;
     final mode = session.flutterMode;
     final isDebug = mode == 'debug';
     final isProfile = mode == 'profile';
     // final isRelease = mode == 'release' || mode == 'jit_release';
     final isFlutter = session.debuggerType?.contains('Flutter') ?? false;
 
-    return TableRow(
+    return MenuBar(
```

Comment:
let's just use a Row instead of a Menu Bar. Visually having the buttons all just be an icon (or text) against a transparent background is probably what we want.

Reply:
The MenuBar seemed like an easy way to get the dropdown for the DevTools buttons. My plan was to make it transparent (I'm assuming that's possible) but I haven't looked too much at the visuals, I wanted to get all the functionality both here and matching in VS Code first since there's no dependency between them for the visuals.

Reply:
We have a shared widget `AnalyticsDropDownButton` that you could use. We likely want to be tracking analytics for all this sidebar stuff so we can measure the usage anyway. Fine to leave as is for now and re-work the UI in a follow up